### PR TITLE
API phpunit 9 support

### PR DESCRIPTION
**Pull request by emteknetnz**
Issue https://github.com/silverstripe/silverstripe-framework/issues/10019

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -10,14 +10,14 @@ env:
 jobs:
   fast_finish: true
   include:
-    - php: 7.1
+    - php: 7.4
```
Comment (maxime-rainville) [inline .travis.yml:13]: We still need to test PHP 7.3 for now.

```suggestion
    - php: 7.3
```

**Review by maxime-rainville — APPROVED** [review]
LGTM

**Review by maxime-rainville — CHANGES_REQUESTED** [review]
Need to update test matrix to test PHP 7.3

File: .travis.yml (hunk 1 above)
Comment (emteknetnz) [reply .travis.yml]: Done

**Review by maxime-rainville — APPROVED** [review]
LGTM

**Pull request merged**